docs/index.md

```diff
@@ -33,29 +33,27 @@ Currently, Redocly OpenAPI CLI supports these features:
 
 ## What makes Redocly OpenAPI CLI different
 
-Unlike other OpenAPI validators, Redocly OpenAPI CLI defines the possible type tree of a valid OpenAPI definition and then traverses it. This approach is very similar to how compilers work, and results in major performance benefits over other approaches.
+As you can see from the previous section, OpenAPI CLI is your all-in-one swiss army knife that can fulfill all your needs when designing, defining, and working with OpenAPI definitions.
```

Comment:
```suggestion
OpenAPI CLI is your all-in-one Swiss army knife that can fulfill your needs when designing, defining, and working with OpenAPI definitions.
```
I think the transition can be perceived negatively. Also, I removed "all" because there is still more to do...

Reply:
Thanks @adamaltman. No objections. Fixed.